### PR TITLE
banlist: add tob functionality, config options

### PR DESCRIPTION
**Pull request by xperiaclash**

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java [hunk 1]
```diff
@@ -232,14 +234,68 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
...
+
+
 	}
 
+
+	@Subscribe
+	public void onGameTick(GameTick event)
+	{
+
+		if (client.getWidget(28, 9) != null)
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java]: This should use widgetinfo api.

File: runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java [hunk 2]
```diff
@@ -36,17 +37,14 @@
...
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.ChatMessageType;
-import net.runelite.api.ClanMember;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.events.ClanMemberJoined;
-import net.runelite.api.events.ConfigChanged;
-import net.runelite.api.events.WidgetHiddenChanged;
-import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.*;
```
Comment (Kyleeld) [inline runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java]: fix wild card import

File: runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java [hunk 3]
```diff
@@ -232,14 +234,68 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
...
 	}
 
+
+	@Subscribe
+	public void onGameTick(GameTick event)
+	{
+
+		if (client.getWidget(28, 9) != null)
+		{
+			Widget raidingParty = client.getWidget(28, 9);
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java]: This should also use widgetinfo api

File: runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java [hunk 4]
```diff
@@ -36,17 +37,14 @@
...
-import net.runelite.api.ChatMessageType;
-import net.runelite.api.ClanMember;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.events.ClanMemberJoined;
-import net.runelite.api.events.ConfigChanged;
-import net.runelite.api.events.WidgetHiddenChanged;
-import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.*;
+import net.runelite.api.events.*;
```
Comment (Kyleeld) [inline runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java]: fix wild card import

**Comment by xperiaclash** [comment]
> 
> 
> Can you fix the tabs in these files?

checkstyle is fine ?

File: runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java [hunk 5]
```diff
@@ -232,14 +231,68 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
...
+					}
+
+
+				}
+			}
+
+
+		}
+
+	}
```
Comment (sdburns1998) [inline runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java]: Can you remove all these empty new lines?

**Pull request merged**